www/docs/API/provider.md

```diff
@@ -227,15 +265,17 @@ Gets the status of a transaction.
 ###### _GetTransactionStatusResponse_
 
 ```
+
 {
-  tx_status: 'NOT_RECEIVED' | 'RECEIVED' | 'PENDING' | 'ACCEPTED_ON_L2' | 'ACCEPTED_ON_L1' | 'REJECTED';
-  block_hash: string;
-  tx_failure_reason?: {
-    tx_id: number;
-    code: string;
-    error_message: string;
-  }
+tx_status: 'NOT_RECEIVED' | 'RECEIVED' | 'PENDING' | 'ACCEPTED_ON_L2' | 'ACCEPTED_ON_L1' | 'REJECTED';
```

Comment:
All of these indentations got messed up it seems?

Reply:
idem

Reply:
the format committed by dhruv is the right one
Seems like these files didn't run through prettier for a while, and as we dont have any recommit hooks it's not required to run prettier before submitting a pr